pkg/installer/filebuilder.go

```diff
@@ -140,6 +144,19 @@ func (o *installerOptions) buildInstallerFileSys(config *apiv1.KubectlStorageOSC
 	}
 	fsData[stosDir] = stosSubDirs
 
+	if config.Spec.Install.InstallLocalPathProvisioner || config.Spec.Uninstall.UninstallLocalPathProvisioner {
```

Comment:
`localPathProvisioner` should be added to `installerOptions` here https://github.com/storageos/kubectl-storageos/blob/921bf0f6b03ca46b04f46419890ce2b20446089b/pkg/installer/filebuilder.go#L17

Then this logic can exist in https://github.com/storageos/kubectl-storageos/blob/921bf0f6b03ca46b04f46419890ce2b20446089b/pkg/installer/installer.go#L191 when building `installerOptions`.

This function (`buildInstallerFileSys`) can then simply check `installerOptions` for what it should include

Ping me if that doesn't make sense 😅

Reply:
Done